ent/schema/attestationpolicy.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 The Archivista Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+)
+
+// Attestation represents an attestation from a witness attestation collection
+type AttestationPolicy struct {
+	ent.Schema
+}
+
+func (AttestationPolicy) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("name").NotEmpty(),
```

Comment:
Ok, we need to agree on this part.
Currently, the Policy does not have a `name`. However, it would make a policy unique, and gitoids would be referenced as versions.

Reply:
I think that adding `name` is the most simple and straight forward solution for this.

Reply:
I added a PR in go-witness
- https://github.com/in-toto/go-witness/pull/209

Reply:
I added a PR in go-witness
- https://github.com/in-toto/go-witness/pull/209